network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/we_found_section.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+<section class="tw-dark tw-my-7 large:tw-my-[120px]">
+  <div class="container tw-max-w-3xl tw-mx-auto">
+    <h4 class="tw-text-lg medium:tw-text-[1.375rem] tw-uppercase tw-font-sans tw-font-bold tw-mb-5">{% trans 'Ultimately, we found:' %}</h4>
+
+    <div class="tw-flex tw-flex-col tw-space-y-5">
+      <div class="tw-flex tw-flex-col medium:tw-flex-row tw-gap-3">
+        <span class="tw-text-4xl tw-text-white">{% trans '01' %}</span>
+        <p class="tw-font-changa tw-text-3xl medium:tw-text-[2.375rem] tw-mb-0">{% blocktrans trimmed %}People overwhelmingly feel that using YouTube’s user controls <span class="tw-text-[#FF3D3D]">does not change</span> their recommendations at all.{% endblocktrans %}</p>
+      </div>
+
+      {# Add special lowercase aria label to stop screen readers from reading as acronym #}
```

Comment:
**Kudos** Thanks for this comment. I was wondering why the value and the label were the same 👍